src/gwt/panmirror/src/editor/package.json

```diff
@@ -46,23 +49,30 @@
     "prosemirror-view": "^1.14.9",
     "react": "^16.13.1",
     "react-dom": "^16.13.1",
+    "react-window": "^1.8.5",
     "tlite": "^0.1.9",
     "typescript": "3.8.3",
     "zenscroll": "^4.0.2"
   },
   "scripts": {
     "format": "prettier --write \"src/**/*.ts\" \"src/**/*.tsx\"",
     "lint": "tslint -c tslint.json 'src/**/*.{ts,tsx}'",
-    "watch": "tsc --watch --noEmit --project './tsconfig.json'"
+    "watch": "tsc --watch --noEmit --project './tsconfig.json'",
+    "generate-symbols": "tsc --types node --lib es2015,dom \"tools/generate-symbols.ts\";node \"tools/generate-symbols.js\"; rm -rf \"tools/generate-symbols.js\""
```

Comment:
The easiest way I know to do this is with `ts-node`.

```
yarn add --dev ts-node
```

Then change the command to:

```
ts-node -T tools/generate-symbols.ts
```

ts-node approximates "a node interpreter that understands typescript", sometimes the seams show through but most of the time it works pretty nicely.

The `-T` is necessary because generate-symbols.ts doesn't seem to pass type check, the `@types/node-fetch` is missing and even after I added that I got some compiler errors. If you add `--strict` to your tsc invocation, you'll see those same errors; fix them and you can drop `-T`.

Reply:
Oh, one of the compiler errors is because

```
import * as fetch from 'node-fetch';
```
should be
```
import fetch from 'node-fetch';
```
Then there are a couple of trivial ones related to null checking.